src/review/logic.py

```diff
@@ -824,24 +824,22 @@ def process_reviewer_csv(path, request, article, form):
         reader = csv.DictReader(csv_file)
         reviewers = []
         for row in reader:
-            try:
-                country = core_models.Country.objects.get(code=row.get('country'))
-            except core_models.Country.DoesNotExist:
-                country = None
-
```

Comment:
This is removed and not handled so this data is lost.

Reply:
Thanks for catching. Again an issue with the old backwards compatibility approach.